modules/nanobind_bazel/2.0.0/MODULE.bazel

```diff
@@ -0,0 +1,14 @@
+module(
+    name = "nanobind_bazel",
+    version = "2.0.0",
+    compatibility_level = 2,
```

Comment:
With Bzlmod, you will only have a single version of nanobind in your graph by default. With different `compatibility_level`s, users with deps that mention both versions would always run into an error instead of being updated to `nanobind@v2` transparently. Instead, they will have to add a `multiple_version_override`.

If you think that the API changes are minor enough that they are unlikely to affect other non-root modules depending on nanobind, it's probably better to keep the same `compatiblity_level`.

Reply:
Thanks, I'm leaning towards reverting the `compatibility_level` change then.

Am I correct in assuming that the algorithm for deciding on that single version in the graph is something like "find all dependencies mentioning nanobind and then go with the highest nanobind version available at the given compatibility level"?

Reply:
Yes, that's correct!